pkg/yurthub/cachemanager/handler_layer.go

```diff
@@ -0,0 +1,117 @@
+package cachemanager
```

Comment:
please add OpenYurt license info

Reply:
License info is already added, thanks for review.